lib/components/SelectionControl/radio/radio-button/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedCircleMdRegularIcon,
+    LabelPairedCircleDotMdFillIcon,
+    LabelPairedCircleInfoMdRegularIcon,
+} from "@deriv/quill-icons";
+import "./radio-button.scss";
+
+type TRadio = {
```

Comment:
In this repo `interface` is used, it's better to stick to it in this case. Also, maybe it's better to extend `input` component props?